app/src/components/csv/CSVErrors.tsx

```diff
@@ -0,0 +1,67 @@
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import AlertBar from 'components/alert/AlertBar';
+import { useMemo } from 'react';
+import { CSVError } from 'utils/csv-utils';
+import { v4 } from 'uuid';
+
+interface CSVErrorsProps {
+  errors: CSVError[];
+}
+
+/**
+ * Returns a stack of CSV errors with information about solutions
+ *
+ * @param {CSVErrorsProps} props
+ * @returns {*}
+ */
+export const CSVErrors = (props: CSVErrorsProps) => {
```

Comment:
nit: Maybe a more descriptive name like `CSVErrorsCardStack`, just something more different than some of the language used previously for CSVErrors etc.